docs/docs/tools/improve.md

```diff
@@ -318,29 +318,46 @@ code_suggestions_self_review_text = "... (your text here) ..."
 
 Under specific conditions, Qodo Merge can auto-approve a PR when a specific comment is invoked, or when the PR meets certain criteria.
 
-To ensure safety, the auto-approval feature is disabled by default. To enable auto-approval, you need to actively set, in a pre-defined _configuration file_, the following:
+To ensure safety, the auto-approval feature is disabled by default.
+To enable auto-approval, you need to actively set, in a pre-defined _configuration file_, by enabling either one the following configs:
 ```toml
 [config]
+enable_manual_approval = true
 enable_auto_approval = true
 ```
-Note that this specific flag cannot be set with a command line argument, only in the configuration file, committed to the repository.
-This ensures that enabling auto-approval is a deliberate decision by the repository owner.
 
-**(1) Auto-approval by commenting**
+!!! note "Notes"
+    - Note that this specific flag cannot be set with a command line argument, only in the configuration file, committed to the repository.
+    - Enabling auto-approval must be a deliberate decision by the repository owner.
+
+1\. **Auto-approval by commenting**
+To enable auto-approval by commenting, set in the configuration file:
```

Comment:
/implement Add an empty line between those two lines to prevent them from becoming a continuous text.